Dockerfile

```diff
@@ -216,7 +216,8 @@ RUN rustup component add \
   rustfmt
 
 # No binary available on Github, have to use cargo install.
-RUN cargo install cargo-deadlinks
+ARG deadlinks_version=0.4.1
```

Comment:
Why not use the latest version? https://github.com/deadlinks/cargo-deadlinks/releases/tag/0.7.0

Reply:
Updated

Reply:
The latest version shows errors in every single file, and it doesn't look like it's a legitimate error:
```
tests ❯ cargo doc ╔════════════════════════
tests ❯ cargo doc ║ ════╡ stdout ╞════
tests ❯ cargo doc ║ Found invalid urls in trusted_database_client_bin/struct.Opt.html:
tests ❯ cargo doc ║ 	Fragment #tymethod.from_clap at trusted_database_client_bin/struct.Opt.html does not exist!
tests ❯ cargo doc ║ Found invalid urls in chat/struct.Router.html:
tests ❯ cargo doc ║ 	Fragment #tymethod.create at chat/struct.Router.html does not exist!
tests ❯ cargo doc ║ 	Fragment #tymethod.handle_command at chat/struct.Router.html does not exist!
tests ❯ cargo doc ║ Found invalid urls in chat/struct.Main.html:
tests ❯ cargo doc ║ 	Fragment #tymethod.handle_command at chat/struct.Main.html does not exist!
tests ❯ cargo doc ║ Found invalid urls in chat/struct.Room.html:
tests ❯ cargo doc ║ 	Fragment #tymethod.invoke at chat/struct.Room.html does not exist!
tests ❯ cargo doc ║ 	Fragment #tymethod.create at chat/struct.Room.html does not exist!
tests ❯ cargo doc ║ Found invalid urls in chat/proto/struct.SendMessageRequest.html:
```

Reply:
Will try to check different versions to see, which one starts this.

Reply:
Version `0.5.0` is the latest, that works